Automattic/wp-calypso
# Domain Management i1: Add icons to expire/renew on column

Closes https://github.com/Automattic/dotcom-forge/issues/3739
## Proposed Changes

* Add appropriate icons to the `Expires/renews` on column

## Testing Instructions



* browse` domains/manage?flags=domains/management`
* Verify the correct icons are shown.

![image](https://github.com/Automattic/wp-calypso/assets/47489215/b5e87eff-2bbf-45b9-bc9f-92bd463cb447)

## Pre-merge Checklist
